src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -600,6 +600,26 @@ private void ScanInterpolatedStringLiteralHoleBalancedText(char endingChar, bool
                                 ScanInterpolatedStringLiteralNestedVerbatimString();
                                 continue;
                             }
+                            else if (lexer.TextWindow.PeekChar(1) == '$' && lexer.TextWindow.PeekChar(2) == '"')
```

Comment:
Thanks for finding and fixing this.

We probably need to add a LangVersion check/diagnostic though (`                       CheckFeatureAvailability(MessageID.IDS_FeatureAltInterpolatedVerbatimStrings)`)